.changeset/rude-wasps-divide.md

```diff
@@ -0,0 +1,12 @@
+---
+'@mysten/dapp-kit': major
```

Comment:
We're pre-major so change this to minor.

Reply:
gotcha, sorry wasn't familiar with the semvar etiquette for this